docs/data/capture/_index.md

```diff
@@ -367,6 +367,18 @@ The following example captures data from the `ReadImage` method of a camera:
 
 {{% /expand%}}
 
+## View captured data
+
+To view captured data for a {{< glossary_tooltip term_id="part" text="machine part" >}}, click on the menu in the top right of its card and select **View captured data**.
```

Comment:
do you want to also mention to view all captured data for a machine you can click on the icon in the top right of the page?

Reply:
actually entirely missed that beforehand until I saw your email